calculator/src/main/java/com/example/calculator/CalculatorApplication.java

```diff
@@ -0,0 +1,23 @@
+package com.example.calculator;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.EnableAspectJAutoProxy;
+
+import javax.persistence.Entity;
+
+@EnableAspectJAutoProxy
```

Comment:
What's the reason of using AspectJ over default spring aop implementation?

Reply:
https://docs.spring.io/spring-framework/docs/current/reference/html/core.html#aop